src/codegen.cpp

```diff
@@ -637,6 +637,15 @@ static AttributeList get_donotdelete_func_attrs(LLVMContext &C)
             None);
 }
 
+static AttributeList get_jlpgcstack_func_attrs(LLVMContext &C)
+{
+    AttributeSet FnAttrs = Attributes(C, {Attribute::InaccessibleMemOnly, Attribute::WillReturn, Attribute::NoUnwind});
+    return AttributeList::get(C,
+            FnAttrs,
+            Attributes(C, {Attribute::NonNull}),
```

Comment:
Maybe add dereferencleable as well, though it might not be useful because it's a pointer to the middle

Reply:
We could change the pointer-to-middle thing, if that matters. Processors should generally deal with small offsets just as well as zero offset for deriving ptls.

Reply:
And if you are adding deref, probably should be adding alignment too